crates/oxc_parser/src/lib.rs

```diff
@@ -148,6 +148,16 @@ impl<'a> Parser<'a> {
     const fn ts_enabled(&self) -> bool {
         self.source_type.is_typescript()
     }
+
+    fn lookahead<F, U>(&mut self, predicate: F) -> U
```

Comment:
Can you move this to 'cursor.rs`? probably next to `try_parse`.

Reply:
Done